pkgs/tools/typesetting/tex/auctex/default.nix

```diff
@@ -1,41 +0,0 @@
-{ lib, stdenv, fetchurl, emacs, texliveBasic, ghostscript }:
-
-let auctex = stdenv.mkDerivation ( rec {
-  # Make this a valid tex(live-new) package;
-  # the pkgs attribute is provided with a hack below.
-  pname = "auctex";
-  version = "13.2";
-  tlType = "run";
-
-  outputs = [ "out" "tex" ];
-
-  src = fetchurl {
-    url = "mirror://gnu/${pname}/${pname}-${version}.tar.gz";
-    hash = "sha256-Hn5AKrz4RmlOuncZklvwlcI+8zpeZgIgHHS2ymCUQDU=";
-  };
-
-  buildInputs = [
-    emacs
-    ghostscript
```

Comment:
The dependencies of this toplevel package and the one inside emacsPackages are different.  Maybe we should update emacsPackages.auctex as well.  According to auctex [manual][2], ghostscript and dvipng seem to be needed at runtime.  Could you verify the need of ghostscript and dvipng and their dependency type?

[2]: https://www.gnu.org/software/auctex/manual/auctex.html#Prerequisites